doc/src/manual/getting-started.md

```diff
@@ -99,11 +99,13 @@ julia [switches] -- [programfile] [args...]
  -h, --help                Print this message
 
  -J, --sysimage <file>     Start up with the given system image file
- --precompiled={yes|no}    Use precompiled code from system image if available
- --compilecache={yes|no}   Enable/disable incremental precompilation of modules
  -H, --home <dir>          Set location of `julia` executable
  --startup-file={yes|no}   Load ~/.juliarc.jl
  --handle-signals={yes|no} Enable or disable Julia's default signal handlers
+ --sysimg-native-code={yes|no}
```

Comment:
spell this `sysimage-native`? (because that's how we spell it in the other option)